eva-accession-import/src/main/resources/application.properties

```diff
@@ -0,0 +1,37 @@
+parameters.assemblyAccession=
+parameters.assemblyName=
+parameters.assemblyReportUrl=
+parameters.taxonomyAccession=
+parameters.chunkSize=100
+#parameters.pageSize=
+parameters.forceRestart=false
+parameters.fasta=
+
+spring.batch.job.names=IMPORT_DBSNP_VARIANTS_JOB
+
+# job repository datasource
+spring.datasource.driver-class-name=
+spring.datasource.url=
+spring.datasource.username=
+spring.datasource.password=
+spring.datasource.tomcat.max-active=3
+spring.jpa.generate-ddl=true
+
+# dbSNP mirror datasource
+dbsnp.datasource.driver-class-name=org.postgresql.Driver
+dbsnp.datasource.url=
+dbsnp.datasource.username=
+dbsnp.datasource.password=
+dbsnp.datasource.tomcat.max-active=
+
+# MongoDB for storing generated accessions
+spring.data.mongodb.host=
+spring.data.mongodb.port=
+spring.data.mongodb.database=
+spring.data.mongodb.username=
+spring.data.mongodb.password=
+spring.data.mongodb.authentication-database=admin
+# is the following property being used?
```

Comment:
Yes, but for writing it should be primary.

Reply:
🎉